www/public/index.html

```diff
@@ -2,116 +2,15 @@
-    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
-    <meta name="theme-color" content="#000b11">
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" />
-    <link rel='stylesheet' href='https://highlightjs.org/static/demo/styles/nord.css'/>
-    <link rel='stylesheet' href='https://necolas.github.io/normalize.css/latest/normalize.css' />
-    <script src="https://js.stripe.com/v3/"></script>
-    <script src="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.16.2/build/highlight.min.js" />
-    <script src="%PUBLIC_URL%/terraform.js" />
-    <script type="text/javascript">
-      hljs.registerLanguage('terraform', window.hljsDefineTerraform);
-      hljs.initHighlightingOnLoad();
-    </script>
-
     <meta name="theme-color" content="#000b11" />
     <meta
       name="description"
-      content="application catalog for multicloud kubernetes"
+      content="One-click deploy for any application"
     />
-    <link rel="apple-touch-icon" href="logo192.png" />
-    <style type="text/css">
-      body {
-        margin: 0;
-        padding: 0;
-        font-family: "Monument", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-          "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-          sans-serif;
-        -webkit-font-smoothing: antialiased;
-        -moz-osx-font-smoothing: grayscale;
-      }
-
-      code {
-        font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-          monospace;
-      }
-
-      pre {
-        margin: 0px 0px;
-      }
-
-      @font-face {
-        font-family: 'Monument';
-        src: url("%PUBLIC_URL%/monument-regular.otf") format("opentype");
-        font-weight: 400;
-      }
-
-      @font-face {
-        font-family: 'Monument';
-        src: url("%PUBLIC_URL%/monument-medium.woff") format("woff");
-        font-weight: 450;
-        size-adjust: 90%;
-      }
-
-      @font-face {
-        font-family: 'Monument';
-        src: url("%PUBLIC_URL%/monument-medium.woff") format("woff");
-        font-weight: 500;
-      }
-
-      @font-face {
-        font-family: 'Monument';
-        src: url("%PUBLIC_URL%/monument-bold.woff") format("woff");
-        font-weight: 500;
-      }
-
-      @font-face {
-        font-family: 'Monument';
-        src: url("%PUBLIC_URL%/monument-regular-italic.woff") format("woff");
-        font-weight: 400;
-        font-style: italic;
-      }
-
-      @font-face {
-        font-family: 'Monument';
-        src: url("%PUBLIC_URL%/monument-medium-italic.woff") format("woff");
-        font-weight: 500;
-        font-style: italic;
-      }
-
-      @font-face {
-        font-family: 'Monument';
-        src: url("%PUBLIC_URL%/monument-bold-italic.woff") format("woff");
-        font-weight: 600;
-        font-style: italic;
-      }
-    </style>
```

Comment:
Seriously who needs that anyway?

Reply:
we do want to have this font loaded as its part of our brand

Reply:
It's not in the design system. The new font is Inter. I'll add it back anyway.

Reply:
Yeah if it isn't in the design system, I think we should change it there too, the monument font is part of our brand system.